recipes/rjieba/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rjieba" %}
+{% set version = "0.1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3ff7396e645c6ebf8c780ac2d585a6c92ddc5e135be83b0da0da9f995ae62f46
+
+build:
+  skip: True  # [osx]
```

Comment:
I don't see a reason to skip `win` or `osx`. Unless there's a good reason, please remove these.